README.md

```diff
@@ -102,6 +102,21 @@ where the version and the revision hash are set accordingly.
 
 Read on if you do want to build from source...
 
+## Wheels
```

Comment:
Nice Table!

Reply:
I agree. There's some legacy stuff above that table in the installation section that needs pruning. It would be great if you could do that bit (including it seems, the stuff on bintray). Your table is much better than most of what else is there. Out of interest, do we still build on every commit?

Reply:
Currently these are only built on: releases, commits in `cibuildwheel` branches.

Reply:
Pruning done